### DISCUSSION TITLE
Community Contributions?

### DISCUSSION THREAD

**Discussion by themontashu** [aderusha/DDD-Printable-Wall-Control-System#51] [Ideas]
I've started designing my own tool holders as the need arises and would be happy to contribute the models. If there's interest and it's not doing to dilute your awesome line I can start uploading stuff as I finish it.

**Comment by aderusha** [comment]
Please do!  Contributions have been coming in pretty regularly and I love seeing how others are using this for themselves!

**Reply by Hercules117** [reply]
What's the best way to upload them? Sorry, not super familiar with GitHub.
